packages/common/src/types/savedCharts.ts

```diff
@@ -233,7 +233,6 @@ export type CreateSavedChartVersion = Omit<
     | 'uuid'
     | 'name'
     | 'updatedAt'
-    | 'projectUuid'
```

Comment:
can we revert this change? `CreateSavedChartVersion` is referenced in many different places